examples/websocket-relay/relay-app/src/lib/workflow.ts

```diff
@@ -132,90 +162,75 @@ export function fail(workflowId: WorkflowId) {
 
 // HANDLER
 
-export async function handleMessage(event: MessageEvent) {
-  const data = await event.data.text();
-
+export async function handleMessage(
+  data: MaybeResult<WorkflowNotification, WorkflowNotificationError>
+) {
   console.log("Received message from server: ", data);
 
-  // Reset ping count on echoed ping or pong from server
-  if (data === "ping" || data === "pong") {
-    activeWorkflowStore.update((store) =>
-      store ? { ...store, failedPingCount: 0 } : null
-    );
+  if (data.error) {
+    // @ts-expect-error received WorkflowNotificationError from Homestar node
+    throw new Error(data.error);
+  }
```

Comment:
what are we ignoring here? you can probably throw `data.error` directly or `new Error('bad message', { cause: data.error })`

Reply:
You're right, nothing left to ignore. Updated to remove the `@ts-expect-error` and throw `data.error` directly.